lib/WP_Auth0_State_Handler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+class WP_Auth0_State_Handler {
+
+  /**
+   * @var string
+   */
+  protected $uniqid = '';
```

Comment:
If this is being set in the `__construct()` method (I guess the class instance constructor) there shouldn't be a need to set it to `''` here.